src/logic/test.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Not sure this should have been committed!

Reply:
Bump! 😄

Reply:
Removed!